src/saving/SaveManagerGUI.cs

```diff
@@ -107,6 +110,7 @@ public override void _Process(float delta)
             Math.Round((float)info.diskSpace / Constants.MEBIBYTE, 2).ToString(CultureInfo.CurrentCulture) + " MiB";
 
         UpdateSelectedCount();
+        UpdateButtonsStatus();
```

Comment:
I think that counting the saves for the cleanup old button should happen here, so that the filesystem doesn't get constantly scanned.